.travis.yml

```diff
@@ -1,12 +1,29 @@
 language: c
 sudo: false
-compiler:
-- clang
-- gcc
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+    - os: linux
+      compiler: clang
+    - os: osx
+      compiler: gcc
```

Comment:
FYI, 'gcc' is aliased to clang/llvm on OS X workers, if you really want gcc, you need to specify gcc-4.8 for instance (I don't remember if it is installed by default, but you can still install it through `brew`)